facebook/hermes
# Add Web Manifest to Hermes Playground

## Summary

This improved the UX of using playground as a PWA-ish standalone app.

FB: Opened as a PR since I got some problem committing `.png` with Eden.



## Test Plan

1. Build the website. 
2. At playground page. "Create shortcut" with "Open as windows" checked.
3. See below screenshot:

![image](https://user-images.githubusercontent.com/5563315/89219582-e275ed00-d584-11ea-8974-8bbe75bb1969.png)

![image](https://user-images.githubusercontent.com/5563315/89219658-0a655080-d585-11ea-8d36-c4555c3bc38b.png)
